test/Main.hs

```diff
@@ -639,6 +640,22 @@ testDBSchema1 step = do
   liftIO $ assertEqual "INSERT into 'witnessed_robbery' table" 3
     (length robberyWitnessIds')
 
+  -- Check that the collation for the 'location' column is the expected "C".
```

Comment:
I wonder if this is redundant - after all, that's exactly what `checkTableStructure` does.

Reply:
OK, you're right... Just enough to basically add a column with a non-default collation then? I'll fixup.

Reply:
Hmmm. I still think that repeating the code from the definition isn't very robust.

How about setting collation en_US for one column and C for another one, then checking that sorting in the first case is case-insensitive and in the second case case-sensitive?

Reply:
Yes, that makes more sense! I added a fixup commit.